server/routes/request.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  loadRequests,
+  createRequest,
+  approveRequest,
+} = require("../controllers/request");
+
+router.route("/").get(protect, loadRequests);
+
+router.route("/create").post(protect, createRequest);
+
+router.route("/approval").put(protect, approveRequest);
```

Comment:
```suggestion
router.route("/:requestId").put(protect, approveRequest);
```
Let's use path params as we're identifying and modifying a single resource :)